lib/EspWire/src/esp_twi.cpp

```diff
@@ -44,6 +44,14 @@ static uint32_t esp_twi_clockStretchLimit;
   #define SCL_LOW()   (GPIO.enable_w1ts = (1 << esp_twi_scl))
   #define SCL_HIGH()  (GPIO.enable_w1tc = (1 << esp_twi_scl))
   #define SCL_READ()  ((GPIO.in & (1 << esp_twi_scl)) != 0)
+#elif defined(ESP32_C3) //where is this defined???
+  //#define SDA_LOW()   (GPIO.enable_w1ts.enable_w1ts = ((uint32_t) 1 << esp_twi_sda)) //Enable SDA (becomes output and since GPO is 0 for the pin, it will pull the line low)
+  static inline void SDA_LOW() { GPIO.enable_w1ts.enable_w1ts = (uint32_t) 1 << esp_twi_sda; GPIO.out_w1tc.out_w1tc = (uint32_t) 1 << esp_twi_sda; }
```

Comment:
`GPIO.enable_w1ts.enable_w1ts` looks weird, are you sure that it is correct? or is it search/replace mispell?

Reply:
the gpio strict is very different here, i just assumed it, might be wrong..
![image](https://github.com/rtlopez/esp-fc/assets/7183506/30fe36a4-3caa-4ffb-a224-86b3dcde4464)